src/orders/components/OrderDraftDetailsProducts/OrderDraftDetailsProducts.tsx

```diff
@@ -164,25 +164,33 @@ const OrderDraftDetailsProducts: React.FC<OrderDraftDetailsProductsProps> = prop
                     initial={{ quantity: line.quantity }}
                     onSubmit={data => onOrderLineChange(line.id, data)}
                   >
-                    {({ change, data, hasChanged, submit }) => (
-                      <DebounceForm
-                        change={change}
-                        submit={hasChanged ? submit : undefined}
-                        time={200}
-                      >
-                        {debounce => (
-                          <TextField
-                            className={classes.quantityField}
-                            fullWidth
-                            name="quantity"
-                            type="number"
-                            value={data.quantity}
-                            onChange={debounce}
-                            onBlur={submit}
-                          />
-                        )}
-                      </DebounceForm>
-                    )}
+                    {({ change, data, hasChanged, submit }) => {
+                      const handleQuantityChange = event => {
+                        if (/^\d*(\.\d+)?$/.test(event.target.value)) {
+                          change(event);
+                        }
+                      };
+
+                      return (
+                        <DebounceForm
+                          change={handleQuantityChange}
+                          submit={hasChanged ? submit : undefined}
+                          time={200}
+                        >
+                          {debounce => (
+                            <TextField
+                              className={classes.quantityField}
+                              fullWidth
+                              name="quantity"
+                              type="number"
+                              value={data.quantity}
+                              onChange={debounce}
+                              onBlur={submit}
+                            />
```

Comment:
```suggestion
                              onBlur={submit}
                              inputProps={{
                                min: 0
                              }}
                            />
```

Or maybe we shouldn't let user drop quantity lower than 1, because it's equal to removing a order line afaik

Reply:
Yep, I guess 1 would be better.